src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1034,10 +1034,10 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
         }
 
         execute_primitive(inst, events);
-        executed_prims++;
```

Comment:
Could you please revert this change as well?

Reply:
oops reverted!